README.md

```diff
@@ -21,11 +21,16 @@ It currently verifies:
 
 
 ## Makefile Targets
-- `make build`: Builds executable
```

Comment:
`make build` is still a valid target, so shouldn't remove it

Reply:
thanks, good catch..